community/openntpd/post-install

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+addgroup -S openntpd
+adduser -S -h /var/empty -G openntpd openntpd
```

Comment:
Maybe add a message for users about the creation of the user + group? Also add that on removal, both will be deleted. I'd much rather we leave this to the user to setup manually (in its entirety) but I'll leave this decision to you. :+1: